lib/still/preprocessor.ex

```diff
@@ -155,22 +149,28 @@ defmodule Still.Preprocessor do
   end
 
   defp preprocessors do
-    preprocessors_to_list(user_defined_preprocessors()) ++
-      preprocessors_to_list(@default_preprocessors)
+    Enum.to_list(user_defined_preprocessors())
+    |> Enum.concat(Enum.to_list(@default_preprocessors))
```

Comment:
They're maps, right? So you don't even need to do `Enum.to_list/1` here.

Reply:
I need, but I can do it the other way around 👍